contrib/ansible/tasks/binaries.yaml

```diff
@@ -1,52 +1,11 @@
 ---
-- name: "Create a directory to download binaries"
-  file: path={{ cri_release_directory }} state=directory
-
 - name: "Get Containerd and CRI-Containerd"
-  get_url:
-    validate_certs: "no"
-    url: "https://storage.googleapis.com/cri-containerd-staging/cri-containerd-{{ cri_containerd_release_version }}.tar.gz"
-    dest: "{{ cri_release_directory }}"
-    mode: 0755
-
-- name: "Unpack Containerd and CRI-Containerd"
   unarchive:
-    src: "{{ cri_release_directory }}cri-containerd-{{ cri_containerd_release_version }}.tar.gz"
-    dest: "{{ cri_release_directory }}"
-    remote_src: yes
-
-- name: "Install the containerd and cri-containerd binaries" 
-  copy:
-    src: "{{ cri_release_directory }}usr/local/bin/{{ item }}"
-    dest: "{{ local_bin_dir }}{{ item }}"
+    src: "https://storage.googleapis.com/cri-containerd-staging/cri-containerd-{{ cri_containerd_release_version }}.tar.gz"
+    dest: "/"
     mode: 0755
```

Comment:
What does 755 means here? Given that we have both binaries and systemd unit files inside the tarball.